src/app/checkboxes/checkboxes.demo.html

```diff
@@ -61,9 +61,51 @@
                 </div>
             </div>
         </div>
+        <div class="form-group">
+            <label>Native Indeterminate Checkbox</label>
+            <div class="checkbox-inline">
+                <input type="checkbox"
+                       id="checkrads_10"
+                       [indeterminate]="nativeIndeterminateState">
+                <label for="checkrads_10">A native indeterminate checkbox.</label>
+            </div>
+        </div>
+    </section>
+</form>
+
+<form #form="ngForm" novalidate>
+    <section class="form-block">
+        <div class="form-group">
+            <label>With ngForm & ngModel</label>
+            <div class="checkbox-inline">
```

Comment:
I am a bit confused by this. I thought that you dont have to wrap the `.checkbox-inline` class on the `clr-checkbox`. `clr-checkbox` takes an input `clrInline` and adds the class on the host itself (`clr-checkbox`) based on the input, no?